ED/protocol.html

```diff
@@ -1067,7 +1067,9 @@ <h4 property="schema:name">Modifying Resources Using N3 Patches</h4>
                       <li id="server-patch-n3-single" rel="spec:requirement" resource="#server-patch-n3-single"><span property="spec:statement">The patch document <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> contain exactly one patch resource, identified by one or more of the triple patterns described above, which all share the same <code>?patch</code> subject.</span></li>
                       <li id="server-patch-n3-simple-type" rel="spec:requirement" resource="#server-patch-n3-type">A patch resource <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> contain a triple <code><span property="spec:statement">?patch rdf:type solid:InsertDeletePatch</span></code>.</li>
                       <li id="server-patch-n3-variables" rel="spec:requirement" resource="#server-patch-n3-variables"><span property="spec:statement">The <code>?insertions</code> and <code>?deletions</code> formulae <span rel="spec:requirementLevel" resource="spec:MUSTNOT">MUST NOT</span> contain variables that do not occur in the <code>?conditions</code> formula.</span></li>
-                      <li id="server-patch-n3-blank-nodes" rel="spec:requirement" resource="#server-patch-n3-blank-nodes"><span property="spec:statement">The <code>?insertions</code> and <code>?deletions</code> formulae <span rel="spec:requirementLevel" resource="spec:MUSTNOT">MUST NOT</span> contain blank nodes.</span></li>
+                      <li id="server-patch-n3-blank-nodes" rel="spec:requirement" resource="#server-patch-n3-blank-nodes"><span property="spec:statement">The <code>?deletions</code> formulae <span rel="spec:requirementLevel" resource="spec:MUSTNOT">MUST NOT</span> contain blank nodes.</span></li>
+                      <li id="server-patch-n3-blank-nodes-syntax" rel="spec:requirement" resource="#server-patch-n3-blank-nodes-syntax"><span property="spec:statement">The same blank node identifier <span rel="spec:requirementLevel" resource="spec:MUSTNOT">MUST NOT</span> appear in more than one scope.</span></li>
```

Comment:
This might more directly convey intentional application rather than just occurrence:

```suggestion
                      <li id="server-patch-n3-blank-nodes-syntax" rel="spec:requirement" resource="#server-patch-n3-blank-nodes-syntax"><span property="spec:statement">A blank node identifier <span rel="spec:requirementLevel" resource="spec:MUSTNOT">MUST NOT</span> be reused in more than one scope.</span></li>
```

That said, the context of this requirement may be unclear to the reader, more specifically the "scope". It might be more clear if it mentions deletes, inserts, the "formula(e)", "patch resource", the "patch document", or something else.

Reply:
Good call - I propose the following clarification 

```suggestion
                      <li id="server-patch-n3-blank-nodes-syntax" rel="spec:requirement" resource="#server-patch-n3-blank-nodes-syntax"><span property="spec:statement">A blank node identifier <span rel="spec:requirementLevel" resource="spec:MUSTNOT">MUST NOT</span> be reused in more than one scope - where the four possible scopes are the patch document scope and the insert, delete and condition formulae.</span></li>
```

Reply:
Closing as this has been picked up in https://github.com/solid/specification/pull/711#discussion_r2019324447